python-api/app/api/v1/openapi/router.py

```
@@ -1,66 +1,68 @@
 
-client = OpenRouterClient()
+client = get_provider("openrouter")
+
 
 @router.post("/completions", response_model=CompletionResponse, tags=["OpenRouter"])
-def create_completion(payload: CompletionRequest):
+def completion(payload: CompletionRequest):
     try:
         return client.completion(payload)
     except Exception as e:
         raise HTTPException(status_code=500, detail=str(e))
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)